package.json

```diff
@@ -102,7 +102,7 @@
     "karma-sourcemap-loader": "0.3.7",
     "karma-webpack": "1.8.0",
     "mocha": "3.0.2",
-    "moment": "2.14.1",
+    "moment": "2.19.3",
```

Comment:
While this satisfies the vulnerability problem, is there a reason we wouldn't just use the latest stable release if we're upgrading it anyway?

Reply:
Yeah, that makes more sense :) 

I'll update it.